main/acle.md

```diff
@@ -9456,6 +9461,709 @@ are named after. All of the functions have external linkage.
+   svuint32x2_t svldnt1[_u32]_x2(svcount_t png, const uint32_t *rn);
+   svuint32x4_t svldnt1[_u32]_x4(svcount_t png, const uint32_t *rn);
+   svuint32x2_t svldnt1_vnum[_u32]_x2(svcount_t png, const uint32_t *rn,
+                                      int64_t vnum);
+   svuint32x4_t svldnt1_vnum[_u32]_x4(svcount_t png, const uint32_t *rn,
+                                      int64_t vnum);
+
+
+   // Variants are also available for _s64 and _f64
+   svuint64x2_t svldnt1[_u64]_x2(svcount_t png, const uint64_t *rn);
+   svuint64x4_t svldnt1[_u64]_x4(svcount_t png, const uint64_t *rn);
+   svuint64x2_t svldnt1_vnum[_u64]_x2(svcount_t png, const uint64_t *rn,
+                                      int64_t vnum);
+   svuint64x4_t svldnt1_vnum[_u64]_x4(svcount_t png, const uint64_t *rn,
+                                      int64_t vnum);
+   ```
+
+#### UMAXQV, SMAXQV, FMAXQV, UMINQV, SMINQV, FMINQV
+
+Max/Min reduction of quadword vector segments.
+
+ ``` c
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _f16, _f32, _f64
+   uint8x16_t svmaxqv[_u8](svbool_t pg, svuint8_t zn);
+   uint8x16_t svminqv[_u8](svbool_t pg, svuint8_t zn);
+   ```
+
+#### FMAXNMQV, FMINNMQV
+
+Max/Min recursive reduction of quadword vector segments.
+
+ ``` c
+   // Variants are also available for  _f32, _f64
+   float16x8_t svmaxnmqv[_f16](svbool_t pg, svfloat16_t zn);
+   float16x8_t svminnmqv[_f16](svbool_t pg, svfloat16_t zn);
+   ```
+
+#### BFMLSLB, BFMLSLT
+
+BFloat16 floating-point multiply-subtract long from single-precision
+(top/bottom).
+
+ ``` c
+
+   // All the intrinsics below are [SVE2.1 or SME2]
+   svfloat32_t svbfmlslb[_f32](svfloat32_t zda, svbfloat16_t zn, svbfloat16_t zm);
+   svfloat32_t svbfmlslb_lane[_f32](svfloat32_t zda, svbfloat16_t zn,
+                                    svbfloat16_t zm, uint64_t imm_idx);
+
+   svfloat32_t svbfmlslt[_f32](svfloat32_t zda, svbfloat16_t zn, svbfloat16_t zm);
+   svfloat32_t svbfmlslt_lane[_f32](svfloat32_t zda, svbfloat16_t zn,
+                                    svbfloat16_t zm, uint64_t imm_idx);
+   ```
+
+#### PEXT
+
+Transform a predicate-as-counter to a predicate (pair).
+
+ ``` c
+   // Variants are also available for _c16, _c32 and _c64 [SVE2.1 or SME2]
+   svbool_t svpext_lane_c8(svcount_t pnn, uint64_t imm);
+   svboolx2_t svpext_lane_c8_x2(svcount_t pnn, uint64_t imm);
+
+   ```
+
+#### PMOV
+
+Move predicate to/from vector. ``pmov`` for element size _u8 or _s8 only accepts
+immediate zero.
+
+ ``` c
+   // Variants are available for:
+   // _s8, _s16, _u16, _s32, _u32, _s64, _u64
+   svbool_t svpmov_lane[_u8](svuint8_t zn, uint64_t imm);
+
+   // Variants are available for:
+   // _s8, _s16, _u16, _s32, _u32, _s64, _u64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svpmov_lane[_u8]_m(svuint8_t zd, svbool_t pn, uint64_t imm);
+   svuint8_t svpmov_lane[_u8]_z(svbool_t pn, uint64_t imm);
+   svuint8_t svpmov_lane[_u8]_x(svbool_t pn, uint64_t imm);
```

Comment:
Hi Carol,
we should change the  _z and _x prototypes to have the type as mandatory and not optional: svpmov_lane_u8